README.md

```diff
@@ -20,7 +20,7 @@ The Subnet EVM runs in a separate process from the main AvalancheGo process and
 ```text
 [v0.7.0] AvalancheGo@v1.12.0-v1.12.1 (Protocol Version: 38)
 [v0.7.1] AvalancheGo@v1.12.2 (Protocol Version: 39)
-[v0.7.2] AvalancheGo@v1.12.2 (Protocol Version: 39)
+[v0.7.2] AvalancheGo@v1.12.2/1.13.0-fuji (Protocol Version: 39)
```

Comment:
🤔 is this a work in progress, will this be changed from `v1.12.2/1.13.0-fuji` to `v1.13.0-fuji` at some point?